src/isPalindrome.js

```diff
@@ -20,6 +20,22 @@
  */
 function isPalindrome(str) {
   // write code here
+  const regStr = /\w/g;
+
+  if (str === '') {
+    return true;
+  }
+
+  const lowerStr = (str.toLowerCase()).match(regStr);
+  const reverseLowerStr = ((str.toLowerCase()).match(regStr)).reverse();
```

Comment:
```suggestion
  const reversed = letters.reverse();
```
а чтобы избежать мутации оригинального массива
```suggestion
  const reversed =[...letters].reverse();
```